CefSharp.WinForms.Example/Handlers/MenuHandler.cs

```diff
@@ -10,16 +10,24 @@ void IContextMenuHandler.OnBeforeContextMenu(IWebBrowser browserControl, IBrowse
         {
             //To disable the menu then call clear
             // model.Clear();
+
+            bool removed = model.Remove(132); // Remove "View Source" option
```

Comment:
I think we probably need to expose the `enum` for commands, so they're a little more user friendly.